src/main/resources/openapi.json

```
@@ -1,314 +1,1279 @@
+                            }
+                        }
+                    },
+                    "required": true
+                },
+                "responses": {
+                    "204": {
+                        "description": "Seek to the beggining performed successfully."
+                    },
+                    "404": {
+                        "description": "The specified consumer instance was not found.",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/Error"
+                                },
+                                "examples": {
+                                    "response": {
+                                        "value": {
+                                            "error_code": 404,
+                                            "message": "The specified consumer instance was not found."
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            },
+            "parameters": [
+                {
+                    "name": "groupid",
+                    "in": "path",
+                    "description": "ID of the consumer group to which the subscribed consumer belongs.",
+                    "required": true,
+                    "schema": {
+                        "type": "string"
+                    }
+                },
+                {
+                    "name": "name",
+                    "in": "path",
+                    "description": "Name of the subscribed consumer.",
+                    "required": true,
+                    "schema": {
+                        "type": "string"
+                    }
+                }
+            ]
         },
-        {
-          "name": "name",
-          "in": "path",
-          "required": true,
-          "schema": {
-            "type": "string"
-          }
-        }
-      ]
-    },
-    "/consumers/{groupid}/instances/{name}/offsets": {
-      "post": {
-        "operationId": "commit",
-        "responses": {
-          "200": {
-            "description": "OK"
-          }
-        }
-      },
-      "parameters": [
-        {
-          "name": "groupid",
-          "in": "path",
-          "required": true,
-          "schema": {
-            "type": "string"
-          }
+        "/consumers/{groupid}/instances/{name}/positions/end": {
+            "post": {
+                "tags": [
+                    "Seek",
+                    "Consumers"
+                ],
+                "description": "Configures a subscribed consumer to seek (and subsequently read from) the offset at the end of one or more given topic partitions. This is the default behavior.",
```

```suggestion
                "description": "Configures a subscribed consumer to seek (and subsequently read from) the offset at the end of one or more of the given topic partitions. This is the default behavior.",
```
"This is the default behavior." I don't think that will make sense to a reader. Its meaning isn't apparent to me.

---

I removed the default sentence.